### ISSUE TITLE
Turn off default submit on forms (accessibility)

### ISSUE THREAD

**Issue by jonibar** [psu-stewardship/etda#92]
When a carriage return occurs when the cursor is positioned on author form fields, the form is submitted.

Turning off the default submit for forms may correct this.  

There are some questions on whether this should be changed....

**Comment by cam156** [comment]
Everything I see online makes me think  we may not want to do this: https://www.smashingmagazine.com/2014/05/mobile-accessibility-why-care-what-can-you-do/

**Comment by jonibar** [comment]
Closing this.
